### ISSUE TITLE
{Linked:Bug11048400} [Adaptive Cards > Designer] - Ensures ARIA attributes are allowed for an element's role for the Role's Checkbox.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8731]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 23H2 (OS Build 25977.1000)
Browser Version: Version 120.0.2186.2 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Run the tool and observe the issue.

**Actual Result:**
Ensures ARIA attributes are not allowed for an roles checkbox element.

**Expected Result:**
Ensures ARIA attributes should be allowed for the checkboxes present under the Role.

**Element path:**
 #__ac-92e0caf3-b44b-4357-915c-59cd8eb5f45d

**Snippet:**
 <div class="ac-input ac-choiceSetInput-multiSelect ac-input-required" tabindex="-1" id="__ac-92e0caf3-b44b-4357-915c-59cd8eb5f45d" aria-required="true" style="width: 100%; min-width: 0px;">

**How to fix:** 
Fix all of the following:
  ARIA attribute is not allowed: aria-required="true"

**User Impact**
It will impact the screen reader user as they will not be able to know the complete information if the ARIA attributes are not allowed for any elements.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)]
(https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)

### Screenshots

![WCAG4 1 2- nsures ARIA attributes are allowed for an element's role for the Role's Checkbox](https://github.com/microsoft/AdaptiveCards/assets/95913748/b1e6644c-a700-49b9-9fdd-d67d9e5d9df7)

![WCAG4 1 2-Ensures ARIA attributes are allowed for an element's role for the checkboxes](https://github.com/microsoft/AdaptiveCards/assets/95913748/f5394741-7c6b-4de6-a252-3beb21666610)

### Card JSON

```JSON
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.5",
    "body": [
        {
            "type": "ColumnSet",
            "columns": [
                {
                    "type": "Column",
                    "width": 2,
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Please enter the below details to create SAP user account request",
                            "weight": "Bolder",
                            "wrap": true
                        },
                        {
                            "type": "Input.ChoiceSet",
                            "id": "Roles",
                            "style": "expanded",
                            "isMultiSelect": true,
                            "isRequired": true,
                            "errorMessage": "Please select the role",
                            "label": "Role",
                            "wrap": true,
                            "choices": [
                                {
                                    "title": "Project Manager",
                                    "value": "Project Manager"
                                },
                                {
                                    "title": "Business Manager",
                                    "value": "Business Manager"
                                },
                                {
                                    "title": "Customer Account Manager",
                                    "value": "Customer Account Manager"
                                },
                                {
                                    "title": "Software Engineer",
                                    "value": "Software Engineer"
                                }
                            ],
                            "value": ""
                        },
                        {
                            "type": "Input.Text",
                            "id": "Justification",
                            "isMultiline": true,
                            "isRequired": true,
                            "errorMessage": "Please provide the business justification",
                            "label": "Business Justification",
                            "value": "",
                            "wrap": true
                        }
                    ]
                }
            ]
        }
    ],
    "actions": [
        {
            "type": "Action.Submit",
            "title": "Submit",
            "data": {
                "id": "SAPUserRequest_Submit"
            }
        }
    ]
}
```

**Comment by vagpt** [comment]
This issue can be originally tracked using URL below.

https://dev.azure.com/MicrosoftIT/OneITVSO/_workitems/edit/11048400

**Comment by anna-dingler** [comment]
Fixed with #8746. Pending website release.

**Comment by vagpt** [comment]
Hi @anna-dingler, 

Thanks for the fixes. Could you please provide us the updated JSON file with the changes so that we can verify this issue at our end as well?

**Comment by vagpt** [comment]
On Bot V1.6 -

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/8af9222a-6409-4009-83ff-ae461bb698b5)

On Teams V1.6 - 

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/22049f71-d461-46d8-a17d-53a995c93d7c)
